app/controllers/replies.rb

```diff
@@ -0,0 +1,14 @@
+class Chitter < Sinatra::Base
+  get '/reply' do
+    session[:peep_id] = params[:peep_id]
+    erb :'replies/reply'
+  end
+
+  post '/reply' do
+    peep = Peep.get(session[:peep_id])
+    reply = Reply.new(text: params[:reply], replier: current_user.username, time: Time.now.asctime)
```

Comment:
Metrics/LineLength: Line is too long. [99/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)